server/app/controllers/user_controller.py

```diff
@@ -0,0 +1,11 @@
+from flask import jsonify, request
+from app.models import project
+from bson import ObjectId
+from . import user_bp
+
+
+# GET Request to retreive all students from the collection
+# @user_bp.route("/addUser", methods=["POST"])
+# def add_user():
+
+ 
```

Comment:
we don't need the user for now, but let's keep the file because we will use it anyway